connectors/aws/src/test/scala/com/raphtory/aws/AwsSpoutTest.scala

```diff
@@ -23,7 +25,7 @@ object AwsSpoutTest extends IOApp {
 
     val spout  = AwsS3Spout(awsS3SpoutBucketName, awsS3SpoutBucketKey)
     val output = AwsS3Sink(awsS3OutputFormatBucketName)
-    val source = Source(spout, LotrGraphBuilder)
+    val source = CSVEdgeListSource(spout)
```

Comment:
Propose we expose these default builders as graph.readEdgeList(path = "...", delimiter = ",") ?

@ljeub-pometry @miratepuffin thoughts?

Reply:
I don't think this scales as we need to support a lot more formats and also the distinction between `stream` and `load`. Not sure the 'Source' in the name is necessary?